backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/Lesson/RequestWithFiltersDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace EasySpeak.Core.Common.DTO.Lesson;
+
+public class RequestWithFiltersDto
```

Comment:
FiltersRequest
FiltersRequestDto if you wish
GetLessonsRequest

Something like this, minimum words like 'with' 'as'...